src/main/java/mate/academy/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -26,6 +32,8 @@ public UserResponseDto save(UserRegistrationRequestDto requestDto)
 
         User user = userMapper.toModel(requestDto);
         user.setEmail(email);
+        user.setPassword(passwordEncoder.encode(user.getPassword()));
+        user.setRoles(roleRepository.findAllByRoles(Set.of(RoleName.USER)));
```

Comment:
If we have hardcoded only `USER` role here why wrap it into a set and do a `findAll...()`? Lets simplify it
```suggestion
        user.setRoles(Collections.singleton(roleRepository.findByName(RoleName.USER)));
```